src/org/sosy_lab/java_smt/test/ModelEvaluationTest.java

```diff
@@ -203,6 +203,7 @@ public void testGetRationalsEvaluation() throws SolverException, InterruptedExce
 
   @Test
   public void testGetBooleansEvaluation() throws SolverException, InterruptedException {
+    // NOTE: Default value for boolean in OpenSMT is 'true'. We patched it to resolve the issue.
```

Comment:
JavaSMT should not change default values of SMT solvers just to its own JUnit test. This might cause overhead in the future when debugging. 
The better solution is to use the default values as given by the SMT solver, and adjust the JUnit test.